### DISCUSSION TITLE
官方的"获取 Token"怎么不能显示二维码了

### DISCUSSION THREAD

**Discussion by GitCourser** [AlistGo/alist#3435] [Q&A] [answered]
这个地址
https://alist.nn.ci/zh/guide/drivers/aliyundrive.html#%E5%88%B7%E6%96%B0%E4%BB%A4%E7%89%8C

点击"获取 Token"后一直是"等待...",看不到二维码

**Comment by RobbieDragon233** [comment]
网页登录获取token挺好用的

**Reply by GitCourser** [reply]
> 网页登录获取token挺好用的

网页上的token不能用啊,只能看文件,不能下载和播放

**Reply by RobbieDragon233** [reply]
嗦嘎！怪不得我升级之后不能正常使用。我换了一个网络后，官网那个token就可以获取了，目前是正常使用的

**Reply by GitCourser** [reply]
刚才可能是网络问题,或者服务器问题,现在这个取token的又能用了,可以正常显示二维码了

**Comment by GitCourser** [comment, marked as answer]
搞好了,我用官网说明中的另一种方法(找手机里日志)找到token了
但有个问题,手机中的token直接填上去后怎么刷新都不行,把容器重启了一次就可以用了

**Reply by Dashan-IZ** [reply]
为什么我的还是添加不了阿里云盘，添加失败

**Reply by GitCourser** [reply]
添加时会检查token,不对就添加不上

**Reply by Dashan-IZ** [reply]
我重复了好几次都不行一直钻圈圈就是添加不了
![image](https://user-images.githubusercontent.com/78125220/219055273-57c5c603-cf52-4f9a-b0b6-e8f078bf7a5c.png)

**Reply by Dashan-IZ** [reply]
为啥我添加不了啊

**Reply by GitCourser** [reply]
你这后面错误写的超时,你看直接上阿里云能不能上,能上应该不会超时
最可能的还是token是错的,填错就添加不上
正确添加后是显示"状态: work", 而不是你这一大串错误提示

**Reply by Dashan-IZ** [reply]
都获取好几次token了，以前这样添加没毛病，今天更新后就不行了，死活添加不上

**Reply by GitCourser** [reply]
你找手机日志文件里的token
或者直接用作者官网上那个取token的二维码,就是最顶上我发的那个地址,刚才不能用,现在已经可以用了

**Reply by Dashan-IZ** [reply]
我发现了，国内网络可以进去，我的是国外服务器，一直验证不了，以前是可以的，我用本地网络一下子就上去了，国外服务器会出现超时现象

**Reply by Dashan-IZ** [reply]
不知道是版本问题，还是这次更新对国外网络给屏蔽掉了
